### PR TITLE
(RK-249) add beaker-hostgenerator and rakefile 

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

File: integration/Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 gem 'beaker', '~> 2.7'
+gem 'rototiller', '= 0.1.0'
+gem 'beaker-hostgenerator', '~> 0.2'
```
Comment (nwolfe) [inline integration/Gemfile]: Can we move this to 0.6.0? 

That release is backwards compatible and has a bunch of new capabilities that will help us get off static config files.

File: integration/Rakefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+desc 'Generate a host configuration used by Beaker'
+rototiller_task :beaker_hostgenerator do |t|
+
+  t.add_command do |c|
+    c.name = 'beaker-hostgenerator'
+    c.argument = '> configs/generated'
+  end
+
+  #this is a hack :(
+  t.add_flag(:name => '', :default => 'centos6-64mdca-64.fa', :override_env => 'TEST_TARGET')
```
Comment (nwolfe) [inline integration/Rakefile:38]: Is this a hack because of beaker-hostgenerator, or something else?

Comment (zreichert) [inline integration/Rakefile:38]: @nwolfe Its a hack with rototiller, I'll have heard a rumor that beaker-hostgenerator is being integrated into Beaker.  If that is true I can remove this task once that happens.

**Comment by scotje** [comment]
👍

**Pull request merged**